src/components/elementView/elementTable.tsx

```diff
@@ -842,7 +858,8 @@ class ElementTable extends React.Component<
                           } else if (
                             col !== "id" &&
                             col !== "network_ports" &&
-                            col !== "comment"
+                            col !== "comment" &&
+                            !isObject(value)
```

Comment:
bug fix -> if the value is an object, ignore it (power connections and mac addresses)